sdk/core/azure-core/inc/azure/core/internal/http/pipeline.hpp

```diff
@@ -47,13 +47,13 @@ namespace Azure { namespace Core { namespace Http { namespace _internal {
     {
       if (policies.size() == 0)
       {
-        throw std::invalid_argument("policies cannot be empty");
+        throw std::invalid_argument("policies cannot be empty"); // LCOV_EXCL_LINE
       }
 
-      m_policies.reserve(policies.size());
+      m_policies.reserve(policies.size()); // LCOV_EXCL_LINE
       for (auto& policy : policies)
       {
-        m_policies.emplace_back(policy->Clone());
+        m_policies.emplace_back(policy->Clone()); // LCOV_EXCL_LINE
       }
```

Comment:
This just looks like a test is missing to cover the ctor with multiple policies. Is that not the case?